### PR TITLE
Add basic usage and enrichment notes to readme

### PR DESCRIPTION
**Pull request by davidstanley01**
This PR adds basic usage and enrichment notes as well as links to the more thorough usage examples in the `examples` directory. 

This is mean to help users discover documentation rather than supplant the existing notebooks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,62 @@
...
-- Go through Watchful API introduction [notebook](https://github.com/Watchfulio/watchful-py/blob/main/examples/api_intro.ipynb).
+
+### Basic Usage
+
+Once you have installed the SDK, import it and begin interacting with it
+
+```python
+import watchful as w
+w.__version__
+#2.3.0
```
Comment (athomasoriginal) [inline README.md:34]: where does this come from? (the `#2.3.0`)

File: README.md [hunk 2]
```diff
@@ -1,27 +1,62 @@
...
+```
+
+> **Note**:
+> For more thorough Watchful API introduction, read through our [introduction notebook](https://github.com/Watchfulio/watchful-py/blob/main/examples/api_intro.ipynb).
+
+### Basic Enrichment
+
+The Watchful SDK aims to make enrichment a straight-forward task. The following step assume a brand new user without any projects.
```
Comment (athomasoriginal) [inline README.md:42]: `assume` -> `assumes`?

File: README.md [hunk 3]
```diff
@@ -1,27 +1,62 @@
...
+
+- Ensure you have the correct version of python, `spacey` models etc
```
Comment (athomasoriginal) [inline README.md:44]: What are the correct versions?

**Review by athomasoriginal — APPROVED** [review]
I think the comments are worth fixing, but you _could_ merge without.

Also, I think we can friendly/polish these up, but my assumption is we want to get _something_ in.  If we want to nail these docs now, I can provide more thoughts.  LMK.

File: README.md (hunk 2 above)
Comment (davidstanley01) [reply README.md:42]: should be `steps assume...`

File: README.md (hunk 3 above)
Comment (shayanjm) [reply README.md:44]: `spacey` should be `spacy` btw

**Comment by davidstanley01** [comment]
@athomasoriginal I addressed those comments. Thank you. And, I think we do want to make a more comprehensive change to make the instructions more friendly but in a later PR when we can be a little more thoughtful. I'm definitely trying to get _**something**_ in for now.

File: README.md (hunk 3 above)
Comment (davidstanley01) [reply README.md]: Removed the word and rewrote the sentence.

**Comment by ryttry** [comment]
Hey folks, thank you for pulling me into this PR. There are a few things I am adding to the PR for intentional reasons:
- Change `brew` to `python` so we don't possibly confuse the user and also to make it feel Pythonic
- Be explicit about enrichment via terminal and Jupyter Notebook, since different type of users have more affinity to one of them than the other
- Other small small changes whenever I am able to catch (I *need to be* trivial/predantic/petty as this is user-facing; I first call myself out in case you realize such detail)

**Review by ryttry — APPROVED** [review]

**Comment by ryttry** [comment]
@davidstanley01 thank you for beefing up the user guide

**Pull request merged**